index.html

```diff
@@ -0,0 +1,102 @@
+            </p>
+            <a href="./signup.html"><span class="add-link">링크 추가하기</span></a>
+            <div class="image-box">
+                <img src="./image/add-link.png" alt="링크의 목록들과 검색기능을 보여주는 이미지" width="100%" height="auto" class="align-center">
+            </div>
+        </section>
+        <section class="justify-center">
+            <div class="column-center">
+                <p class="title">
+                    <span class="font-gradient2">원하는 링크</span>를<br>
+                    저장하세요
+                </p>
+                <p class="content">
+                    나중에 읽고 싶은 글, 다시 보고 싶은 영상<br>
+                    사고 싶은 옷 기억하고 싶은 모든 것을<br>
+                    한 공간에 저장하세요
+                </p>
+            </div>
+            <div>
+                <img src="./image/save-link.png" alt="원하는 링크를 저장하는 것을 보여주는 이미지" width="100%" height="auto">
+            </div>
+        </section>
+        <section class="justify-center">
+            <div>
+                <img src="./image/change-name.png" alt="폴더 이름 변경하는 것을 보여주는 이미지" width="100%" height="auto">
+            </div>
+            <div class="column-center">
+                <p class="title">
+                    링크를 폴더로<br>
+                    <span class="font-gradient3">관리</span>하세요
+                </p>
+                <p class="content">
+                    나만의 폴더를 무제한으로 만들고<br>
+                    다양하게 활용할 수 있습니다.
+                </p>
+            </div>
+        </section>
+        <section class="justify-center">
+            <div class="column-center">
+                <p class="title">
+                    저장한 링크를<br>
+                    <span class="font-gradient4">공유</span>해 보세요.
+                </p>
+                <p class="content">
+                    여러 링크를 폴더에 담고 공유할 수 있습니다.<br>
+                    가족, 친구, 동료들에게 쉽고 빠르게 링크를<br>
+                    공유해 보세요.
+                </p>
+            </div>
+            <div>
+                <img src="./image/share.png" alt="폴더 공유하는 것을 보여주는 이미지" width="100%" height="auto">
+            </div>
+        </section>
+        <section class="justify-center">
+            <div>
+                <img src="./image/search.png" alt="저장한 링크를 검색하는 것을 보여주는 이미지" width="100%" height="auto">
+            </div>
+            <div class="column-center">
+                <p class="title">
+                    저장한 링크를<br>
+                    <span class="font-gradient5">검색</span>해 보세요
+                </p>
+                <p class="content">
+                    중요한 정보들을 검색으로 쉽게 찾아보세요.
+                </p>
+            </div>
+        </section>
+    </div>
+    <footer class="footer-style">
+        <span class="codeit-logo">©codeit - 2023</span> 
+        <span>
+            <a href="./privacy.html">Privacy Policy</a>
+            <a href="./faq.html">FAQ</a>
+        </span>
+        <span class="footer-icons">
+            <a href="https://www.facebook.com" target="_blank"><img src="./image/icon-facebook.png"></a>
```

Comment:
a 태그에 `target="_blank"` 속성을 사용하실 때는 rel 속성도 함께 사용해주시는게 좋습니다! [참고](https://velog.io/@neori/HTML-a%EC%97%90-targetblank%EC%99%80-relnoreferrer-noopener%EB%A5%BC-%ED%95%A8%EA%BB%98-%EC%82%AC%EC%9A%A9%ED%95%B4%EC%95%BC-%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0)

```suggestion
            <a href="https://www.facebook.com" target="_blank" rel="noreferrer noopener"><img src="./image/icon-facebook.png"></a>
```